src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10650,7 +10673,13 @@ namespace HVACUnitarySystem {
 			} // ELSE ! No Moisture Load
 		} // No Heating/Cooling Load
 
-		MSHPMassFlowRateLow = CompOffMassFlow; // these need to be set for multi-speed coils
+		if ( MultiOrVarSpeedHeatCoil( UnitarySysNum ) && ( HeatingLoad && HeatSpeedNum == 1 ) ) {
+			MSHPMassFlowRateLow = CompOnMassFlow * PartLoadRatio; // proportional to PLR when speed = 1
+		} else if ( MultiOrVarSpeedCoolCoil( UnitarySysNum ) && ( CoolingLoad && CoolSpeedNum == 1 ) ) {
+			MSHPMassFlowRateLow = CompOnMassFlow * PartLoadRatio; // proportional to PLR when speed = 1
+		} else {
+			MSHPMassFlowRateLow = CompOffMassFlow; // these need to be set for multi-speed coils
+		}
```

Comment:
Here is a special case where MSHPMassFlowRateLow is set based on PLR if multispeed, and still using old method if  variable speed.